packages/core/src/App/Containers/VerificationModal/verification-modal.scss

```diff
@@ -10,7 +10,7 @@
 
     &__main-container {
         margin: 0 auto;
-        max-width: 68.2rem;
+        max-width: 84rem !important;
```

Comment:
please don't use important but instead change the exact classname you need / re-order classnames to achieve the desired result. There are only 2 files where it's used only: verification-modal.scss & account-common.scss

Reply:
this is fixed. 👍🏼